lambdas/certexpiryreminder/build.gradle

```diff
@@ -0,0 +1,115 @@
+import org.gradle.api.tasks.testing.logging.TestExceptionFormat
+import org.gradle.api.tasks.testing.logging.TestLogEvent
+
+plugins {
+	id "java"
+	id "idea"
+	id "jacoco"
+	id 'io.freefair.aspectj.post-compile-weaving' version '6.3.0'
+}
+
+// Exclusion needed as http clients are picked from the classpath rather than from client builders.
+// The follow error occurs if exclusion removed -
+// "Multiple HTTP implementations were found on the classpath.
+// To avoid non-deterministic loading implementations, please explicitly provide an HTTP client via the client builders,
+// set the software.amazon.awssdk.http.service.impl system property with the FQCN of the HTTP service to use as the default,
+// or remove all but one HTTP implementation from the classpath: software.amazon.awssdk.core.exception.SdkClientException"
+// TODO switch to client builders and remove exclusion - see passport-v1 ClientFactoryService
+configurations.configureEach {
+	exclude group:"software.amazon.awssdk", module: "apache-client"
+}
+
+dependencies {
+	implementation project(":lib"),
+			"uk.gov.account:cri-common-lib:${dependencyVersions.cri_common_lib_version}",
+			"com.nimbusds:oauth2-oidc-sdk:${dependencyVersions.nimbusds_oauth_version}",
+			"com.nimbusds:nimbus-jose-jwt:${dependencyVersions.nimbusds_jwt_version}",
+			"com.amazonaws:aws-lambda-java-core:${dependencyVersions.aws_lambda_core_version}",
```

Comment:
Do we need all these dependencies in here the lambda functionality should be quite limited

Reply:
No I've inherited these from another lambda will update to reflect whats needed